internal/service/securitylake/data_lake_test.go

```diff
@@ -344,6 +390,31 @@ const testAccDataLakeConfigConfig_base = `
 data "aws_caller_identity" "current" {}
 data "aws_partition" "current" {}
 
+resource "aws_iam_role" "meta_store_manager_v1" {
+  name               = "AmazonSecurityLakeMetaStoreManagerV1"
+  path               = "/service-role/"
+  assume_role_policy = <<POLICY
+{
+  "Version": "2012-10-17",
+  "Statement": [{
+    "Sid": "AllowLambda",
+    "Effect": "Allow",
+    "Principal": {
+      "Service": [
+        "lambda.amazonaws.com"
+      ]
+    },
+    "Action": "sts:AssumeRole"
+  }]
+}
+POLICY
+}
+
+resource "aws_iam_role_policy_attachment" "datalake_v1" {
+  role       = aws_iam_role.meta_store_manager_v1.name
+  policy_arn = "arn:${data.aws_partition.current.partition}:iam::aws:policy/service-role/AmazonSecurityLakeMetastoreManager"
+}
+
```

Comment:
Since this Role is only used in the `testAccDataLakeConfig_metaStore` and `testAccDataLakeConfig_metaStoreUpdate`, it would be better to only include it in those configurations, rather than for all tests